crates/re_viewer_context/src/gpu_bridge/tensor_to_gpu.rs

```diff
@@ -210,17 +210,14 @@ pub fn class_id_tensor_to_gpu(
         "Cannot apply annotations to tensor of shape {:?}",
         tensor.shape
     );
-    anyhow::ensure!(
-        tensor.dtype().is_integer(),
-        "Only integer tensors can be annotated"
-    );
 
-    let (min, max) = tensor_stats
+    let (_, mut max) = tensor_stats
         .range
         .ok_or_else(|| anyhow::anyhow!("compressed_tensor!?"))?;
-    anyhow::ensure!(0.0 <= min, "Negative class id");
```

Comment:
Why was this check removed?
Should we support negative class ids?
Have you tested it?